apps/api/src/billing/controllers/usage/usage.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { singleton } from "tsyringe";
+
+import { UsageHistoryResponse, UsageHistoryStats } from "@src/billing/http-schemas/usage.schema";
+import { UsageService } from "@src/billing/services/usage/usage.service";
+
+@singleton()
+export class UsageController {
+  constructor(private readonly usageService: UsageService) {}
+
+  async getHistory(address: string, startDate: string, endDate: string): Promise<UsageHistoryResponse> {
```

Comment:
We should memoize these endpoints to something like few seconds, maybe 15 seconds